spec/layout_spec.rb

```diff
@@ -3,8 +3,74 @@
 require 'spec_helper.rb'
 
 RSpec.describe 'Home page' do
+  before { get '/' }
+  let(:page) { Nokogiri::HTML(last_response.body) }
+
   it 'loads the page' do
-    get '/'
     expect(last_response).to be_ok
   end
+
+  it 'loads the donation form' do
+    expect(last_response.body).to include('<form')
```

Comment:
Why not `page` here?

Reply:
I think it was just because those are hardcoded. We will come back to them when we hit those use cases and have to actually add selectors.